ec2/InstanceType.dhall

```diff
@@ -314,153 +315,377 @@ let alternatives =
             → λ(instanceType : InstanceType)
             → merge
                 { c5-large =
-                      [ InstanceType.c5d-large, InstanceType.c5n-large ]
-                    : List InstanceType
+                    let cross-instances =
+                          [ InstanceType.m4-large
+                          , InstanceType.m5-large
```

Comment:
`m5a-large`, `m5d-large`, `m5ad-large`, `m5n-large`, `m5dn-large`?

Reply:
Don't think it should be here (at least not at this point, maybe if we feel it is needed we'll add it in the future).